internal/admin/server_go119_test.go

```diff
@@ -0,0 +1,37 @@
+//go:build go1.19
+// +build go1.19
+
+package admin
+
+var expectedMetrics = []string{
+	"go_gc_duration_seconds",
+	"go_goroutines",
+	"go_info",
+	"go_memstats_alloc_bytes",
+	"go_memstats_alloc_bytes_total",
+	"go_memstats_buck_hash_sys_bytes",
+	"go_memstats_frees_total",
+	"go_memstats_gc_sys_bytes",
+	"go_memstats_heap_alloc_bytes",
+	"go_memstats_heap_idle_bytes",
+	"go_memstats_heap_inuse_bytes",
+	"go_memstats_heap_objects",
+	"go_memstats_heap_released_bytes",
+	"go_memstats_heap_sys_bytes",
+	"go_memstats_last_gc_time_seconds",
+	"go_memstats_lookups_total",
+	"go_memstats_mallocs_total",
+	"go_memstats_mcache_inuse_bytes",
+	"go_memstats_mcache_sys_bytes",
+	"go_memstats_mspan_inuse_bytes",
+	"go_memstats_mspan_sys_bytes",
+	"go_memstats_next_gc_bytes",
+	"go_memstats_other_sys_bytes",
+	"go_memstats_stack_inuse_bytes",
+	"go_memstats_stack_sys_bytes",
+	"go_memstats_sys_bytes",
+	"go_sched_gomaxprocs_threads",
```

Comment:
:no_bell: this one seems to be the diff between 1.18 and 1.19?

Reply:
Yup, new feature in Go 1.19.

Reply:
I think I would skip the build tags and just do

```
expectedMetrics := ...

if !strings.HasPrefix(runtime.Version(), "go1.18") {
  expectedMetrics = append(...)
}
```

Or you could use https://pkg.go.dev/golang.org/x/mod/semver if you want to be a bit more precise.

Reply:
So, I thought about this a bit.
* Go doesn't actually use semver, so `semver.Compare()` is going to be messy.
* The build tags match Go version >=, so its a bit future-proof .

I think we need to keep this testing the way it is. (This is how upstream Prometheus client_golang does testing)